docs/source/using/experimental-features.adoc

```diff
@@ -87,3 +87,31 @@ PS> minishift.exe start `
 Be sure to specify a valid gateway and nameserver.
 Failing to do so will result in connectivity issues.
 ====
+
+
+[[set-fixed-ip]]
+== Set fixed IP Address
+
+While the VM gets assigned an IP address using DHCP, most hypervisors do not handle extending the lease time of the assignment.
+This can cause issues on restarts which assigns a new IP address to the instance. While the VM will start, it will conflict
```

Comment:
Suggestion: 
Most hypervisors do not support extending the lease time when the IP is assigned using DHCP. 
This might lead to a new IP getting assigned to the VM after a restart as it will conflict with the security certificates generated for the old IP.  This will make Minishift completely unusable.